### DISCUSSION TITLE
Audio+image plugin

### DISCUSSION THREAD

**Discussion by kassyec** [jspsych/jsPsych#2988] [Q&A]
Is there a way to have an image and audio stimulus being presented at the same time for a slider/button response plugin? For example is there a way to edit the audio-button response-plugin to have an image on the screen during the trial?

**Comment by ChristopheBossens** [comment]
Have you tried using the prompt property of the audio-button-response plugin? It can contain HTML so you should be able to write something like:

```
var trial = {
    type: jsPsychAudioButtonResponse,
    stimulus: 'sound/tone.mp3',
    choices: ['Low', 'High'],
    prompt: "<img src='my_image.jpg'>"
};
```
